ebusd/CHANGELOG.md

```
@@ -1,4 +1,9 @@
 <!-- https://developers.home-assistant.io/docs/add-ons/presentation#keeping-a-changelog -->
+
+## version: 24.1.1
```

Markdownlint (0.37.0)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)